### PR TITLE
Upgrade all projects to richie v2.0.1

### PR DESCRIPTION
**Pull request by sampaccoud**
## Purpose

We want to upgrade all sites to [richie 2.0.1](https://github.com/openfun/richie/releases/tag/v2.0.1).

## Proposal

I ran the helper command:

```bash
bin/upgrade 2.0.1 -c
```

- [x] Updated each project's settings according to what has changed
- [x] Bumped dependencies to their latest version

### CODE REVIEW

**Review by jbpenrath — COMMENTED** [review]
We have also to update settings and this PR is the right moment I think.

**Review by jbpenrath — COMMENTED** [review]
You should add an entry in site's changelog.

**Comment by sampaccoud** [comment]
@jbpenrath I upgraded to richie 2.0.1 and adapted all settings. Ready for further review...

File: sites/cnfpt/requirements/base.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ django-storages==1.10.1
 dockerflow==2020.10.0
 gunicorn==20.0.4
 psycopg2-binary==2.8.6
-richie==2.0.0-beta.22
+richie==2.0.1
 sentry-sdk==0.19.1
```
Comment (jbpenrath) [inline sites/cnfpt/requirements/base.txt:10]: Relevant to synchronize package versions with richie ?

- **factory-boy:** `3.2.0`
- **sentry-dsk**: `0.19.5`

**Review by jbpenrath — APPROVED** [review]
LGTM

File: sites/cnfpt/requirements/base.txt (hunk 1 above)
Comment (sampaccoud) [reply sites/cnfpt/requirements/base.txt:10]: yes good idea

File: sites/demo/requirements/base.txt [hunk 2]
```diff
@@ -1,9 +1,9 @@
 boto3==1.9.96
```
Comment (jbpenrath) [inline sites/demo/requirements/base.txt:1]: You updated boto3 to `1.16.53` in cnfpt.

File: sites/funcampus/requirements/base.txt [hunk 3]
```diff
@@ -2,12 +2,12 @@ boto3==1.9.96
 django-configurations==2.2
```
Comment (jbpenrath) [inline sites/funcampus/requirements/base.txt:2]: Ditto boto3

File: sites/funcorporate/requirements/base.txt [hunk 4]
```diff
@@ -2,12 +2,12 @@ boto3==1.9.96
 django-configurations==2.2
```
Comment (jbpenrath) [inline sites/funcorporate/requirements/base.txt:2]: ditto boto3

File: sites/funmooc/requirements/base.txt [hunk 5]
```diff
@@ -2,9 +2,9 @@ boto3==1.9.96
 django-configurations==2.2
```
Comment (jbpenrath) [inline sites/funmooc/requirements/base.txt:2]: ditto boto3

File: sites/demo/requirements/base.txt (hunk 2 above)
Comment (sampaccoud) [reply sites/demo/requirements/base.txt]: sorry daddy! done

**Pull request merged**